### ISSUE TITLE
[BUG] Swimming acts weird when hitting "climbable blocks"

### ISSUE THREAD

**Issue by RadicalRadio** [Tria-Studio/TRIA-Issues#49]
**Describe the bug**
I'm _not really sure_ how to describe the bug, but I'll try the best I can: when your arms hit blocks that could be climbable outside of liquids it tends to mess with animations, messing with hitboxes and making it harder to go through gaps

**To Reproduce**
1. Load in a map that uses swimming
2. Enter a liquid that isn't lava (for obvious reasons)
3. Make your arms hit that block
4. See that the animations become weird and hitboxes are different

**Expected behavior**
Normally, you should be able to swim through or simply get blocked by it without having animations stop/bug.

**Screenshots**

https://user-images.githubusercontent.com/104731777/180666392-f1c5ce9c-4e6f-4932-bda0-121fea6ffac3.mp4

https://user-images.githubusercontent.com/104731777/180666425-f2a852c1-2896-4ed1-b003-f36eb8c184bf.mp4

**Hardware Information**
 - Device: practically any device
 - OS: any OS
 - Version: TRIA.os v0.8 (stable)

**Additional context**
If you need any further explaination, just notify me as this bug was *really* difficult to explain lol

**Comment by Ethan76167** [comment]
1 word
what

**Comment by Ethan76167** [comment]
@RadicalRadio does this still happen

**Comment by RadicalRadio** [comment]
I'd have to check when I'm able to

**Comment by RadicalRadio** [comment]
This still happens, yeah

**Comment by Ethan76167** [comment]
> This still happens, yeah
> 
> 

Can you make a map to reproduce this. I need to test a potential fix

**Comment by Ethan76167** [comment]
> oh it must enable the climing state

wdym

**Comment by RadicalRadio** [comment]
I tried another test, but this time I climbed on a truss, climbed down until I was submerged in the water, and then tried moving and it looks like the same thing happens.

https://user-images.githubusercontent.com/104731777/183270470-775b0cc7-80f5-4339-ae50-7726e4d929a9.mp4

I think that @GlowingUmbreon has a point. Maybe the game thinks you are in a climbing state and then that causes the animations to mess up?
